WalletWasabi.Tests/UnitTests/WabiSabi/Client/ArenaClientTests.cs

```diff
@@ -57,10 +58,14 @@ public async Task RegisterInputAsyncTest()
 				Money.Coins(.25m)
 			};
 
+			var inputWeight = 4 * Constants.P2wpkhInputVirtualSize;
+			var inputRemainingWeights = new[] { protocolMaxWeightPerAlice - inputWeight };
```

Comment:
FWIW i think `MaxWeightPerAlice` (or perhaps `WeightAllocationPerAlice`) is a much clearer name than `round.RegistrableWeightCredentials` which is what it's currently known as